examples/java/spring-boot/src/test/java/uk/gov/api/springboot/config/CorrelationIdFilterIntegrationTest.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.api.springboot.config;
+
+import io.restassured.module.jsv.JsonSchemaValidator;
+import me.jvt.spring.ContentNegotiator;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.UUID;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.header;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class CorrelationIdFilterIntegrationTest {
+
+  @Autowired
+  private MockMvc mockMvc;
+
+  @MockBean
+  private ContentNegotiator contentNegotiator;
```

Comment:
I think we may want to have this as a real version, instead of the mock, so we can be clear it works